core/src/test/java/org/apache/iceberg/TestMergeAppend.java

```diff
@@ -1375,47 +1294,45 @@ public void testManifestEntryFieldIdsForChangedPartitionSpecForV1Table() {
             .next();
     Types.NestedField field =
         ((PartitionData) entry.file().partition()).getPartitionType().fields().get(0);
-    Assert.assertEquals(1000, field.fieldId());
-    Assert.assertEquals("id_bucket", field.name());
+    assertThat(field.fieldId()).isEqualTo(1000);
+    assertThat(field.name()).isEqualTo("id_bucket");
     field = ((PartitionData) entry.file().partition()).getPartitionType().fields().get(1);
-    Assert.assertEquals(1001, field.fieldId());
-    Assert.assertEquals("data_bucket", field.name());
+    assertThat(field.fieldId()).isEqualTo(1001);
+    assertThat(field.name()).isEqualTo("data_bucket");
 
     entry =
         ManifestFiles.read(committedSnapshot.allManifests(table.io()).get(1), FILE_IO)
             .entries()
             .iterator()
             .next();
     field = ((PartitionData) entry.file().partition()).getPartitionType().fields().get(0);
-    Assert.assertEquals(1000, field.fieldId());
-    Assert.assertEquals("data_bucket", field.name());
+    assertThat(field.fieldId()).isEqualTo(1000);
+    assertThat(field.name()).isEqualTo("data_bucket");
   }
 
-  @Test
+  @TestTemplate
   public void testDefaultPartitionSummaries() {
     table.newFastAppend().appendFile(FILE_A).commit();
 
     Set<String> partitionSummaryKeys =
         table.currentSnapshot().summary().keySet().stream()
             .filter(key -> key.startsWith(SnapshotSummary.CHANGED_PARTITION_PREFIX))
             .collect(Collectors.toSet());
-    Assert.assertEquals(
-        "Should include no partition summaries by default", 0, partitionSummaryKeys.size());
+    assertThat(partitionSummaryKeys).isEmpty();
 
     String summariesIncluded =
         table
             .currentSnapshot()
             .summary()
             .getOrDefault(SnapshotSummary.PARTITION_SUMMARY_PROP, "false");
-    Assert.assertEquals(
-        "Should not set partition-summaries-included to true", "false", summariesIncluded);
+    assertThat(summariesIncluded).isEqualTo("false");
 
     String changedPartitions =
         table.currentSnapshot().summary().get(SnapshotSummary.CHANGED_PARTITION_COUNT_PROP);
-    Assert.assertEquals("Should set changed partition count", "1", changedPartitions);
+    assertThat(changedPartitions).isEqualTo("1");
```

Comment:
same as I mentioned in the other test file

Reply:
sure, thanks